prowler/lib/check/check.py

```diff
@@ -483,18 +492,34 @@ def run_fixer(check_findings: list):
                     check_module_path = f"prowler.providers.{findings[0].check_metadata.Provider}.services.{findings[0].check_metadata.ServiceName}.{check}.{check}_fixer"
                     lib = import_check(check_module_path)
                     fixer = getattr(lib, "fixer")
-                except AttributeError:
+                except ModuleNotFoundError:
                     logger.error(f"Fixer method not implemented for check {check}")
                 else:
                     print(
                         f"\nFixing fails for check {Fore.YELLOW}{check}{Style.RESET_ALL}..."
                     )
                     for finding in findings:
                         if finding.status == "FAIL":
-                            print(
-                                f"\t{orange_color}FIXING{Style.RESET_ALL} {finding.region}... {(Fore.GREEN + 'DONE') if fixer(finding.region) else (Fore.RED + 'ERROR')}{Style.RESET_ALL}"
-                            )
-                    print()
+                            # Check if fixer has region as argument to check if it is a region specific fixer
+                            if "region" in fixer.__code__.co_varnames:
+                                print(
+                                    f"\t{orange_color}FIXING{Style.RESET_ALL} {finding.region}... "
+                                )
+                                if fixer(finding.region):
+                                    fixed_findings += 1
+                                    print(f"\t{Fore.GREEN}DONE{Style.RESET_ALL}")
+                                else:
+                                    print(f"\t{Fore.RED}ERROR{Style.RESET_ALL}")
```

Comment:
Can you move this to a function not to repeat it? Something like `print_fixer_result`.